src/frontend/src/utils/iiConnection.ts

```diff
@@ -393,7 +394,8 @@ export class Connection {
     const actor = await this.createActor();
     return await actor.add_tentative_device(userNumber, {
       ...device,
-      origin: window?.origin === undefined ? [] : [window.origin],
+      // TODO: use toNullable
```

Comment:
leftover?

Reply:
No todo for a separate PR.

Reply:
Then please track this separately, not in the code